configs/4.0/ffigen.yaml

```diff
@@ -108,3 +108,11 @@ dart-bool: true
 
 typedef-map:
   "size_t": "Int32"
+
+structs:
+  rename:
+    '_(.*)': '$1'
```

Comment:
I think double quotes (") should be used by local convention of this file.

Also consider removing blank lines between top-level elements for consistency.

Reply:
Replace all double quotes to single quote. And keep the blank lines between top-level elements for readability, but I add some more blank lines for the consistency.